README.md

```diff
@@ -42,6 +48,14 @@ The service runs in the background consuming messages from Kafka. The messages a
 | VAULT_PATH                   | secret/shared/psk                 | The path where the psks will be stored in vault
 | DOWNLOAD_SERVICE_URL         | http://localhost:23600            | The public address of the download service
 
+Notes:
+
+1. Ignored unless using TLS (i.e. `KAFKA_SEC_PROTO` has a value enabling TLS)
```

Comment:
can you use [this](https://github.com/ONSdigital/dp-filter-api/pull/214/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R49) as the template for these notes, please

Reply:
Done